modules/shared/src/main/scala/org/tessellation/schema/aci/StateChannelOutput.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.schema.aci
+
+import org.tessellation.schema.address.Address
+
+sealed trait StateChannelOutput {
```

Comment:
Maybe we should have this special case called `StateChannelSnapshotOutput` as I guess it reflects better the restriction that we make here?